build/test-integration.sh

```diff
@@ -6,21 +6,12 @@
 
 set -e
 
-declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
```

Comment:
We did it this way so that this script could be run from any directory. With this change, we are supposed to run from within `build` directory as `__variables.sh` would only be available there.